pkg/oc/lib/describe/describer.go

```diff
@@ -77,7 +82,7 @@ func describerMap(clientConfig *rest.Config, kclient kclientset.Interface, host
 	if err != nil {
 		glog.V(1).Info(err)
 	}
-	appsClient, err := appsclient.NewForConfig(clientConfig)
+	appsClient, err := appstypedclient.NewForConfig(clientConfig)
```

Comment:
hm, what about upstream describers? I believe some of those make use of the `pkg/printers/internalversion` package.
Since we do not use the printers/internalversion package in kubectl to the extent that we used to (with humanreadable printing) due to the introduction of server-side printing, maybe the best path forward for describers would be something similar to server-side printing?

Reply:
@juanvallejo not super familiar with server side printing, but in general I think describers are not that generic and are very custom for some resources. I think for origin resources, as we continue moving forward with external clients and external types, we will likely convert our describers to external... I would expect that if kubectl is also taking the external path, they will do the same... 

/cc @soltysh

Reply:
At least in the scope of this PR, does switching the describer func for deploymentconfigs affect the `oc describe` command?